lib/ansible/modules/cloud/cloudstack/cs_zone_info.py

```diff
@@ -0,0 +1,203 @@
+#!/usr/bin/python
+# -*- coding: utf-8 -*-
+#
+# Copyright (c) 2016, René Moser <mail@renemoser.net>
+# GNU General Public License v3.0+ (see COPYING or https://www.gnu.org/licenses/gpl-3.0.txt)
+
+from __future__ import absolute_import, division, print_function
+__metaclass__ = type
+
+ANSIBLE_METADATA = {'metadata_version': '1.1',
+                    'status': ['stableinterface'],
+                    'supported_by': 'community'}
+
+
+DOCUMENTATION = '''
+---
+module: cs_zone_info
+short_description: Gathering information about zones from Apache CloudStack based clouds.
+description:
+  - Gathering information from the API of a zone.
+version_added: '2.9'
+author: René Moser (@resmo)
+options:
+  zone:
+    description:
+      - Name of the zone.
+      - If not specified, all zones are returned
+    type: str
+    aliases: [ name ]
+extends_documentation_fragment: cloudstack
+'''
+
+EXAMPLES = '''
+- name: Gather information from a zone
+  cs_zone_info:
+    zone: ch-gva-1
+  register: zone
+  delegate_to: localhost
```

Comment:
Perhaps we could add an example to show how to list zones (without filtering).

Reply:
Added one, will push once tests ran through.